ibis/backends/duckdb/tests/test_register.py

```diff
@@ -177,13 +177,48 @@ def pgurl():  # pragma: no cover
     os.environ.get("DUCKDB_POSTGRES") is None, reason="avoiding CI shenanigans"
 )
 def test_read_postgres(con, pgurl):  # pragma: no cover
+    # we don't run this test in CI, only locally, to avoid bringing a postgres
+    # container up just for this test.  To run locally set env variable to True
+    # and once a postgres container is up run the test.
     table = con.read_postgres(
         f"postgres://{pgurl.username}:{pgurl.password}@{pgurl.host}:{pgurl.port}",
         table_name="duckdb_test",
     )
     assert table.count().execute()
 
 
+@pytest.fixture(scope="session")
+def mysqlurl():  # pragma: no cover
+    mysqlcon = ibis.mysql.connect(user="ibis", password="ibis", database="ibis_testing")
+
+    df = pd.DataFrame({"x": [1.0, 2.0, 3.0, 1.0], "y": ["a", "b", "c", "a"]})
+    s = ibis.schema(dict(x="float64", y="str"))
+
+    mysqlcon.create_table("duckdb_test", df, schema=s, overwrite=True)
+    yield mysqlcon.con
+    mysqlcon.drop_table("duckdb_test", force=True)
+
+
+@pytest.mark.skipif(
+    os.environ.get("DUCKDB_MYSQL") is None, reason="avoiding CI shenanigans"
+)
```

Comment:
I'll put up a PR to do this with markers. That way we can avoid mystery environment variables.

Reply:
Hm nevermind, this doesn't seem worth the effort right now.